README.dev.rst

```diff
@@ -32,6 +32,14 @@ like so:
 
     howfairis https://github.com/<owner>/<repo>
 
+Docker
+---------------
+To build the image run:
+docker build -t fairsoftware/howfairis:VERSION .
```

Comment:
```suggestion
docker build -t fairsoftware/howfairis:VERSION .
```

maybe not use VERSION as the default, because then you have to also use the VERSION when running